guides/common/modules/proc_reset-custom-ssl-certificate-to-default-self-signed-certificate-on-project.adoc

```diff
@@ -0,0 +1,22 @@
+[id="reset-custom-ssl-certificate-to-default-self-signed-certificate-on-{project-context}_{context}"]
+= Reset {customssl} certificate to default self-signed certificate on {ProjectServer}
+
+.Procedure
+* Reset the {customssl} certificate to default self-signed certificate:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --certs-reset
+----
+
+.Verification
+Ensure the following parameters in `/etc/foreman-installer/scenarios.d/satellite-answers.yaml` have no values:
```

Comment:
This needs some branding. I'd suggest "{project-context}" but for orcharhino, the file is called "katello-answers.yaml".

Reply:
Added a separate condition for `orcharhino` :+1: